locales/fr/fr.json

```diff
@@ -326,12 +326,12 @@
     "Liberia": "Libéria",
     "Libyan Arab Jamahiriya": "Libye",
     "Liechtenstein": "Liechtenstein",
-    "Light": "Light",
+    "Light": "Claire",
```

Comment:
L'adjectif "light" est communément utilisé dans sa version masculine. 
Un mode "clair"

Reply:
effectivement

Reply:
I have fixed it

Reply:
Besoin de préciser que dans le contexte de Laravel Nova, les mots « Dark », « Light » et « System » sont utilisés pour sélectionner la palette de couleurs du site ? Ce contexte vous aidera-t-il à choisir la bonne option ?